atest/run.py

```diff
@@ -208,6 +206,7 @@ def execute_tests(interpreter, browser, rf_options, grid, event_firing):
     options.extend([opt.format(browser=browser) for opt in ROBOT_OPTIONS])
     if rf_options:
         options += rf_options
+    options += ["--exclude", f"known issue {browser.replace('headless', '')}"]
```

Comment:
Do you think this is fine or should this be done the same way as  `browser` is inserted into `ROBOT_OPTIONS`?

Reply:
looks good imho

Reply:
I don't think we can assume it would be headless. Instead I would treat headless as separate. As I write this I was hoping that I could get some ease to having to write both Known Issue Chrome and Known Issue Headless Chrome each and every time. But realized this is the same assuming that a chrome issue affect both headed and headless. 

Would a method be that if it affect only a headed or headless operation of the browser to say `Known Issue Chrome headless only`? Can we post process this with the current toolset or is this a bit more work?

Reply:
@Brownies Would you want to separate this question of known issues from the critical fix? Not saying you have too but I was assuming that the non critical is what is failing the test against Robot Framework version > 5.0.  If it is then separating this might allow us to resolve that and check newer RF versions while working through this question of known issues. Just a thought and open to differing opinions on this.

Reply:
I'm quite sure handling headed and headless issues separately with a single tag will require some more effort.

I don't really want to separate this from the `--noncritical` fix because this is a replacement for that functionality. I can remove it if you really want to but it will cause tests with known issues to be executed unless users manually `--exclude` them. This might not break Github actions because I have had tests tagged as `Known Issue Chrome` actually pass.

Reply:
I may have understood this wrong. I will relook at it sometime over the next few days.